src/dataframe/dataframe.jl

```diff
@@ -180,15 +180,11 @@ function DataFrame(pairs::Pair{<:AbstractString,<:Any}...; makeunique::Bool=fals
                      copycols=copycols)
 end
 
-# these two are needed as a workaround Tables.jl dispatch
+# this is needed as a workaround for Tables.jl dispatch
 DataFrame(pairs::AbstractVector{<:Pair}; makeunique::Bool=false,
           copycols::Bool=true) =
     DataFrame(pairs..., makeunique=makeunique, copycols=copycols)
```

Comment:
Can we get ride of this one also?
All it is doing is calling `DataFrame(pairs...)` 
can't the user do that themselves?

Reply:
I guess this one is handy so you can do:
```
DataFrame(colnames .=> cols)
```
etc

Reply:
Exactly, in other places we got recurring requests from the users to keep allowing vectors, where one could splat them (e.g. in  `select` etc.) as it is easy to construct vectors programatically.

Reply:
Yes splatting should be avoided when possible as it forces compilation of methods specialized on the number of arguments.